ecl/dedicated_hypervisor/v1/vcenter.py

```diff
@@ -68,3 +71,19 @@ def update(self, session, vcenter_id, password, license_id):
         )
         self._translate_response(resp, has_body=True)
         return self
+
+
+class Link_local_addresses(VCenter):
```

Comment:
```suggestion
class LinkLocalAddresses(resource2.Resource):
```
クラス名の付け方（大文字小文字のルール）が既存の他のクラスと整合していません。
クラス名を変えると _proxy.pyのほうの記述にも影響するので注意してください。
また VCenterクラスを継承する必然性がありません。

Reply:
ご指摘のとおり修正いたします。